src/server/rollbar.js

```diff
@@ -30,6 +30,15 @@ function Rollbar(options, client) {
   }
 }
 
+Rollbar.instance = null;
```

Comment:
Yeah I was actually just getting around to doing that because the way I did it with instance doesn't quite work that well.